e2e/_suites/metricbeat/metricbeat_test.go

```diff
@@ -363,18 +371,11 @@ func (mts *MetricbeatTestSuite) installedUsingConfiguration(configuration string
 
 	metricbeatVersion = e2e.CheckPRVersion(metricbeatVersion, metricbeatVersionBase)
 
-	// use master branch for snapshots
-	tag := "v" + metricbeatVersion
```

Comment:
I think this is not needed anymore, as we will never build an URL for a tag, always branches or commits.

Reply:
Would that still be the case we if stick to testing the way the front facing user documentation suggests?

Reply:
Did I forget updating a link? 🤦‍♀️  Please let me know it and will update the PR tomorrow